cocos/particle/particle-system.ts

```diff
@@ -949,6 +949,15 @@ export class ParticleSystem extends ModelRenderer {
             this._isStopped = false;
         }
 
+        if (!this._isEmitting) {
```

Comment:
I don't think it's should be reset when `isEmitting` is false. `isEmitting` will be changed sometime.